### PR TITLE
Restyle side menu

### PR DESCRIPTION
**Pull request by dominik-zeglen**
I want to merge this change because it implements new sidebar design.

**PR intended to be tested with API branch:** master

### Screenshots
Screenshot 2020-09-07 at 17 52 02
Screenshot 2020-09-07 at 17 52 20
Screenshot 2020-09-07 at 17 52 39

### Pull Request Checklist

1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] Translated strings are extracted.
1. [ ] Number of API calls is optimized.
1. [x] The changes are tested.
1. [x] Data-test are added for new elements.
1. [ ] Type definitions are up to date.
1. [x] Changes are mentioned in the changelog.

### Test environment config

API_URI=https://master.staging.saleor.rocks/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [CU-7utb1g Rewrite sidebar menu](https://app.clickup.com/t/7utb1g)

**Review by AlicjaSzu — APPROVED** [review]

**Review by orzechdev — CHANGES_REQUESTED** [review]
1. When I open submenu, I'm getting:
Zrzut ekranu 2020-09-8 o 11 46 19
2. When I go to categories and open submenu, under which is search bar, I cannot click on submenu item, instead I am only able to activate search bar:
Zrzut ekranu 2020-09-8 o 12 01 13

File: src/components/SideBar/utils.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  }
+
+  return location.split("?")[0] === orderDraftListUrl().split("?")[0] &&
+    menuItem.url.split("?")[0] === orderListUrl().split("?")[0]
+    ? false
+    : !!matchPath(location.split("?")[0], {
+        exact: menuItem.url.split("?")[0] === "/",
+        path: menuItem.url.split("?")[0]
+      });
+}
```
[the remark this answers is not in the document]
Comment (dominik-zeglen) [reply src/components/SideBar/utils.ts:21]: Mindlessly copied some code, but yeah, it looks way nicer

**Comment by dominik-zeglen** [comment]
All fixed 👌

**Review by orzechdev — APPROVED** [review]

**Review by krzysztofwolski — APPROVED** [review]

**Review by krzysztofwolski — CHANGES_REQUESTED** [review]
image
Hamburger icon appears. Seems to be leftover component.

**Comment by dominik-zeglen** [comment]
Mobile version will be added in another PR @krzysztofwolski

**Review by krzysztofwolski — APPROVED** [review]

**Pull request merged**